sql/core/src/test/scala/org/apache/spark/sql/PostgreSQLDialectQuerySuite.scala

```diff
@@ -39,4 +39,15 @@ class PostgreSQLDialectQuerySuite extends QueryTest with SharedSparkSession {
       intercept[IllegalArgumentException](sql(s"select cast('$input' as boolean)").collect())
     }
   }
+
+  test("cast to timestamp") {
+    Seq(1, 0.1, 1.toDouble, 5.toFloat, true, 3.toByte, 4.toShort) foreach { value =>
+        val actualResult = intercept[AnalysisException](
+          sql(s"SELECT CAST(${value} AS timestamp)")
+        ).getMessage
+        val expectedResult = s"cannot cast type ${value.getClass} to timestamp"
+        assert(actualResult.contains(expectedResult))
+    }
+  }
```

Comment:
Can you move these tests to `SQLQueryTestSuite`,e.g., `input/postgreSQL/cast.sql`?

Reply:
I have moved these test cases. `cast.sql.out` needs to be updated.

Reply:
Need to delete this test case.